### PR TITLE
Displays 'No current vacancies' if no job roles listed

### PR DESCRIPTION
**Pull request by DennisTsiang**
If roles is nil then it displays 
- 'No current vacancies' as the title
- "We don't currently have any team openings" underneath.

### CODE REVIEW

File: source/assets/stylesheets/styles/_jobs_page_tile.scss [hunk 1]
```diff
@@ -110,3 +110,10 @@
     min-height: 106px;
   }
 }
+
+.jobs-page-tile-no-roles__paragraph {
```
Comment (neilvanbeinum) [inline source/assets/stylesheets/styles/_jobs_page_tile.scss]: We should stick to the block name in the file for this rule (or make a new file for the new block). How do you feel about `.jobs-page-tile__no-roles-paragraph`?

**Review by neilvanbeinum — CHANGES_REQUESTED** [review]
Mostly looks good!

I noticed that at mobile widths there's no margin between this section and the next. I think a margin-bottom on the paragraph element for these widths should do the trick.

**Comment by DennisTsiang** [comment]
Amended those changes and squashed them to the previous commit

File: source/jobs.erb [hunk 2]
```diff
@@ -22,18 +22,18 @@ unboxed_meeting_image:
...
+#        Our developers thrive on collaboration with customers and delivering
+#        digital products.
+#      link: /jobs/developer
+#      image: /assets/images/jobs/developer.jpg
+#    - name: Designer
+#      text: >-
+#        We embrace design thinking and collaboration, as part of our
+#        multi-skilled agile teams.
+#      link: /jobs/designer
+#      image: /assets/images/jobs/designer.jpg
```
Comment (neilvanbeinum) [inline source/jobs.erb]: We'll need to uncomment these :)

Comment (DennisTsiang) [reply source/jobs.erb]: Ah sorry, I left that in during testing

**Review by neilvanbeinum — APPROVED** [review]

**Comment by neilvanbeinum** [comment]
Thanks for taking the time to do the class rename! I think that's an improvement 👍

**Pull request merged**